props/saber_BC_buttons.h

```diff
@@ -0,0 +1,1092 @@
+/*
+saber_BC_buttons.h
```

Comment:
Sure, but I looked at all of the other prop files and don't see anything like that. What would I need that for?

Reply:
Any pointers here?

Reply:
Fredrik, when you get a chance, this one has me stumped.

Reply:
It can be pretty simple, something like:

Copyright 2021 by:
... list of authors here
Distributed under the terms of the GNU General Public License v3.

Reply:
Updated.